arduino-ide.sls

```diff
@@ -0,0 +1,22 @@
+{% set versions = {
+  '2.3.2': '2.3.2.0',
+  '2.3.1': '2.3.1.0',
+  '2.3.0': '2.3.0.0',
+  '2.2.1': '2.2.1.0',
+  '2.2.0': '2.2.0.0',
+  '2.1.1': '2.1.1.0'
+  }
+%}
```

Comment:
Please look at `salt-minion.sls` or `firefox.sls` where we are settling on using inline yaml for version lists.
e.g. `{% load_yaml as versions -%}` and it seems to me that `winversion` is redundant; at least at the moment you could just append `.0` in the yaml below.

Reply:
Thank you for your review. The new commit should apply all your suggestions.